src/legacy/core_plugins/console/np_ready/public/lib/token_iterator/token_iterator.test.ts

```diff
@@ -0,0 +1,176 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { TokenIteratorImpl } from './token_iterator';
+import { Position, Token, TokensProvider } from '../../interfaces';
+
+const mockTokensProviderFactory = (tokenMtx: Token[][]): TokensProvider => {
+  return {
+    getTokens(lineNumber: number): Token[] | null {
+      return tokenMtx[lineNumber - 1] || null;
+    },
+    getTokenAt(pos: Position): Token | null {
+      return null as any;
+    },
+  };
+};
+
+describe('Token Iterator', () => {
+  // describe('detecting current position', () => {
```

Comment:
Did you intend to leave these test cases commented out?

Reply:
Yep, this spec is actually from another branch I haven't made a PR for. I'd like to get around to reviving this test coverage.